internal/kgateway/translator/irtranslator/route.go

```diff
@@ -196,6 +200,31 @@ func applyRouteTimeout(ctx context.Context, route *envoy_config_route_v3.Route,
 	}
 }
 
+// applyRouteRetry applies the retry policy to the route.
+func applyRouteRetry(route *envoy_config_route_v3.Route, retry *ir.RetryIR) {
+	retryPolicy := &envoy_config_route_v3.RetryPolicy{
+		NumRetries: &wrapperspb.UInt32Value{Value: 3}, // default
```

Comment:
```suggestion
		NumRetries: &wrapperspb.UInt32Value{Value: 1}, // default
```
1 has worked pretty well previously to strike a balance on most workloads that need some amount of retrying.